scripts/generate_contributors.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/*
+ * This script fetches a maximum of PAGE_COUNT * 30 contributors and writes
+ * their usernames and url to the Contributors.md file.
+ */
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const PAGE_COUNT = 4;
+const FILE_NAME = "Contributors.md";
```

Comment:
do we need the absolute path for this file?

Reply:
This script is meant to be run from the root. I copied the logic from the other script but I guess we can make it more robust.

Reply:
I made it so it can only be executed from root.